db/src/main/kotlin/edu/artic/db/AppConverters.kt

```diff
@@ -0,0 +1,236 @@
+                        )
+                )
+            }
+            return@let audioTranslationListAdapter!!
+        }
+    }
+
+    private fun getGeneralInfoTranslationListAdapter() : JsonAdapter<List<ArticGeneralInfo.Translation>>{
+        return generalInfoTranslationListAdapter.let {
+            if(it == null) {
+                generalInfoTranslationListAdapter = moshi.adapter(
+                        Types.newParameterizedType(
+                                List::class.java,
+                                ArticGeneralInfo.Translation::class.java
+                        )
+                )
+            }
+            return@let generalInfoTranslationListAdapter!!
+        }
+    }
+
+    private fun getAudioComentaryListAdapter() : JsonAdapter<List<AudioCommentaryObject>>{
+        return audioCommentaryObjectListAdapter.let {
+            if(it == null) {
+                audioCommentaryObjectListAdapter = moshi.adapter(
+                        Types.newParameterizedType(
+                                List::class.java,
+                                AudioCommentaryObject::class.java
+                        )
+                )
+            }
+            return@let audioCommentaryObjectListAdapter!!
+        }
+    }
+
+    private fun getTourTranslationListAdapter() : JsonAdapter<List<ArticTour.Translation>>{
+        return tourTranslationListAdapter.let {
+            if(it == null) {
+                tourTranslationListAdapter = moshi.adapter(
+                        Types.newParameterizedType(
+                                List::class.java,
+                                ArticTour.Translation::class.java
+                        )
+                )
+            }
+            return@let tourTranslationListAdapter!!
+        }
+    }
+
+    private fun getTourStopListAdapter() : JsonAdapter<List<ArticTour.TourStop>>{
+        return tourStopListAdapter.let {
+            if(it == null) {
+                tourStopListAdapter = moshi.adapter(
+                        Types.newParameterizedType(
+                                List::class.java,
+                                ArticTour.TourStop::class.java
+                        )
+                )
+            }
+            return@let tourStopListAdapter!!
+        }
+    }
+
+    private fun getTourCategoryListAdapter() : JsonAdapter<List<ArticTourCategory>>{
+        return tourCategoryListAdapter.let {
+            if(it == null) {
+                tourCategoryListAdapter = moshi.adapter(
+                        Types.newParameterizedType(
+                                List::class.java,
+                                ArticTourCategory::class.java
+                        )
+                )
+            }
+            return@let tourCategoryListAdapter!!
+        }
+    }
+
+    @TypeConverter
+    fun stringToList(data: String?): List<String> {
+        return data.let {
```

Comment:
lot's of `data.let` repeated code in this file. make it an inline method that specifes the adapter for these specific objects.
something like:
```
private inline fun <T> safeList(adapterGetter: () -> JsonAdapter<List<T>>): List<T> {
        return data?.let { adapterGetter().fromJson(it)  }
            ?: emptyList()
}
```
something like this would suffice everywhere this is repeated.